scripts/azureml-assets/azureml/assets/util/sku_utils.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""SKU utils."""
+
+import json
+import requests
+from azureml.assets.util import logger
+from azureml.assets.util.util import retry
+from azure.identity import AzureCliCredential
+
+
+all_sku_details = None
+SKU_DETAILS_URI = (
```

Comment:
Do we have something specific to the SKUs AzureML supports?

Reply:
Not sure, but I was not able to find any open and generic API not linked to subscription and region.

Reply:
Created an internal thread to know more on this. Thanks